go-mode.el

```diff
@@ -1391,32 +1391,46 @@ If IGNORE-CASE is non-nil, the comparison is case-insensitive."
 
 
 (defun go-packages ()
-  (funcall go-packages-function))
```

Comment:
I don't think we want to change the implementation of (go-packages). This allows users to override the default behavior easily by setting the go-packages-function. It would be surprising if we suddenly started manipulating the return results of that based on the current workspace.